packages/api/src/hooks/useDxtradeAccountsList.ts

```diff
@@ -15,18 +15,20 @@ const useDxtradeAccountsList = () => {
         () =>
             dxtrade_accounts?.trading_platform_accounts?.map(account => ({
                 ...account,
-                display_balance: Intl.NumberFormat(authorize_data?.preferred_language || 'en-US', {
+                /** The balance of the account in currency format. */
+                display_balance: `${Intl.NumberFormat(authorize_data?.preferred_language || 'en-US', {
                     minimumFractionDigits: 2,
                     maximumFractionDigits: 2,
                     minimumIntegerDigits: 1,
-                }).format(account?.balance || 0),
+                }).format(account?.balance || 0)} ${account?.currency || 'USD'}`,
             })),
         [authorize_data?.preferred_language, dxtrade_accounts?.trading_platform_accounts]
     );
 
     return {
         /** List of all created Deriv X accounts */
         data: modified_dxtrade_accounts,
+        ...rest,
```

Comment:
I tend to avoid non-explicit parameters/fields whenever possible, to make it easier to read. Just like "any".